config-common/src/lib.rs

```diff
@@ -4,11 +4,15 @@
 #![warn(clippy::all, clippy::pedantic)]
 #![allow(
     clippy::default_trait_access,
+    clippy::large_enum_variant,
     clippy::missing_errors_doc,
     clippy::module_name_repetitions
 )]
 
 pub mod error;
+
+pub mod super_config;
+
```

Comment:
any reason for this whitespace?

Reply:
I was doing a refactor and forgot to finish it. Making `mod error` not pub and instead making the `Error` a pub re-export. Done now.